scripts/dartagnan

```diff
@@ -76,7 +86,7 @@ if [ -z "${target}" ]; then
 		*imm*)
         		target=imm
                 ;;
-		*linux-kernel*)
+		*lkmm*)
```

Comment:
This will not work. The cat file is called `linux-kernel.cat`, not `lkmm.cat`

Reply:
Ohh, that was a mistake! Sorry

Reply:
Fixed!